src/TDB2.cpp

```diff
@@ -57,6 +57,15 @@ TDB2::TDB2 ()
 ////////////////////////////////////////////////////////////////////////////////
 void TDB2::open_replica (const std::string& location, bool create_if_missing)
 {
+  File pending_data = File (location + "/pending.data");
+  if (pending_data.exists()) {
+    Color warning = Color (Context::getContext ().config.get ("color.warning"));
+    std::cout << warning.colorize (
+      format ("Found existing '.data' files in {1}", location)) << "\n";
+    std::cout << "  Taskwarrior's storage format changed in 3.0.\n";
+    std::cout << "  See https://github.com/GothenburgBitFactory/taskwarrior/releases.\n";
+    std::cout << "  Remove the files to silence this warning.\n\n";
```

Comment:
```suggestion
```

I wouldn't make this suggestion in a warning message because a user might delete these files without exporting first. Really they should probably move them elsewhere for a temporary backup but IMO just pointing them towards longer form documentation is all a warning message should aspire to do.

Reply:
Thanks, good advice!